sdk/servicebus/service-bus/src/sender.ts

```diff
@@ -199,10 +215,34 @@ export class SenderImpl implements Sender {
     return this._isClosed || this._context.isClosed;
   }
 
-  async send(message: ServiceBusMessage, options?: OperationOptions): Promise<void> {
+  async send(message: ServiceBusMessage, options?: OperationOptions): Promise<void>;
+  async send(messages: ServiceBusMessage[], options?: OperationOptions): Promise<void>;
+  async send(
+    messageOrMessages: ServiceBusMessage | ServiceBusMessage[],
+    options?: OperationOptions
+  ): Promise<void> {
     this._throwIfSenderOrConnectionClosed();
-    throwTypeErrorIfParameterMissing(this._context.namespace.connectionId, "message", message);
-    return this._sender.send(message, options);
+
+    if (Array.isArray(messageOrMessages)) {
+      const batch = await this.createBatch(options);
+
+      for (const message of messageOrMessages) {
+        if (!batch.tryAdd(message)) {
+          throw new Error(
+            "Messages were too big to fit in a single batch. Remove some messages and try again or use createBatch() and sendBatch(), which gives more fine-grained control."
```

Comment:
If we send too big a message, then the service throws an error. Can we re-use that here?

Reply:
That seems like a nice way to do it, especially since we don't support "partial" send. I'll fix it up.

Reply:
Done (and test removed).

Reply:
I dont follow... I still see a custom error being thrown here when tryAdd() returns false

Reply:
Done (and test removed) for real this time. Forgot to push.

Reply:
Well, now we see that I'm about to undo this :)

Reply:
Added back.